auto_causality/optimiser.py

```diff
@@ -117,10 +116,10 @@ def __init__(
         self._settings["try_init_configs"] = try_init_configs
 
         self.metric = metric
-        if metric not in ["erupt", "norm_erupt", "qini", "auc", "ate", "r_score"]:
+        if metric not in ["erupt", "norm_erupt", "qini", "auc", "ate", "r_score", "iv_energy_score"]:
```

Comment:
Can we please just call it something like "energy_distance"? It makes sense just as well for regular CATE estimation